R-package/src/Makevars.win.in

```diff
@@ -0,0 +1,60 @@
+CXX_STD = CXX11
+
+PKGROOT=.
+
+LGB_CPPFLAGS = \
+    @LGB_CPPFLAGS@ \
+    -DUSE_SOCKET \
+    -DLGB_R_BUILD \
+    -static-libstdc++ \
```

Comment:
I remember this flag was needed to workaround conflicts with `conda`. Maybe try to not add it at the first attempt? Refer to #899, https://github.com/dmlc/xgboost/issues/977.

Reply:
oh interesting! Yeah I just had put it here trying to mimic CMakeLists. I can try without it

Reply:
I just pushed https://github.com/microsoft/LightGBM/pull/3188/commits/53108bd3503f16ad1b7a2003f29ad017944ab108, which removes this. Let's see if it works ok.